runtime/drivers/clickhouse/olap.go

```diff
@@ -107,9 +107,13 @@ func (c *connection) Execute(ctx context.Context, stmt *drivers.Statement) (res
 		return nil, err
 	}
 
-	stmt.Query += "\n SETTINGS cast_keep_nullable = 1, join_use_nulls = 1, session_timezone = 'UTC'"
-	if c.config.EnableCache {
-		stmt.Query += ", use_query_cache = 1"
+	if c.config.SettingsOverride != "" {
+		stmt.Query += "\n SETTINGS " + c.config.SettingsOverride
```

Comment:
Do we need a way to mention achieve - do not override any settings i.e. No settings to be configured ?